contracts/cw20-dao-crowdfund/src/contract.rs

```diff
@@ -88,9 +97,15 @@ pub fn instantiate(
 
     let msg = SubMsg::reply_on_success(birth_msg, INSTANTIATE_FUNDING_TOKEN_REPLY_ID);
 
-    Ok(Response::default()
+    let mut response = Response::default()
```

Comment:
Instead of making this mutable how about returning a response object from the above if statement which calls `take_public_payment` and then adding attributes to that? Not a logic issue here but any time I see mut it's normally a red flag. Here's an example of this: https://github.com/DA0-DA0/dao-contracts/blob/cc84cbea97ee7a7e3d0091a1b4fd2c2007d45d07/contracts/cw-govmod-single/src/contract.rs#L180-L198